google-cloud-pubsub/OVERVIEW.md

```diff
@@ -316,6 +316,36 @@ received_messages = sub.pull
 sub.modify_ack_deadline 120, received_messages
 ```
 
+## Minimizing API calls before receiving and acknowledging messages
+
+A subscription object can be created making any API calls by providing the
+`skip_lookup` argument to {Google::Cloud::Pubsub::Project#subscription
+Project#subscription} or {Google::Cloud::Pubsub::Topic#subscription
+Topic#subscription}. A subscriber object can also be created without an API call
+by providing the `deadline` optional argument to
+{Google::Cloud::Pubsub::Subscription#listen Subscription#listen}:
+
```

Comment:
Can you add the following paragraph here:

```
Skipping API calls may be used to avoid `Google::Cloud::PermissionDeniedError`
if your account has limited access to the Pub/Sub API. In particular, the role
`roles/pubsub.subscriber` does not have the permission `pubsub.subscriptions.get`,
which is required to retrieve a subscription resource. See [Access Control - 
Roles](https://cloud.google.com/pubsub/docs/access-control#tbl_roles)
for the complete list of Pub/Sub roles and permissions.
```

Reply:
Added this paragraph after the code example. PTAL.